src/main/java/ui/windows/game/GameDisplay.java

```diff
@@ -42,6 +49,7 @@ public GameDisplay(GameDelegator gameDelegator, CardDelegator cardDelegator) {
         this.drawDelegator = gameDelegator.getDrawDelegator();
         this.cardDelegator = cardDelegator;
         this.currentCard = new DrawnCard(Suit.HEART, Rank.ACE);
+        this.playerInfo = new Object[][]{{"", 0}, {"", 0}, {"", 0}, {"", 0}, {"", 0}, {"", 0}};
```

Comment:
Maybe you could make the 0s all Strings and have this as String[][], you'll have to do conversions back and forth between String and Integer though. It's your call though.

Reply:
That's a good idea, that way the 0s won't show up when there's no players to take up that row in the game. I will try to implement this now. :)